rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -4697,6 +4748,34 @@ declare export class ScriptPubkey {
    */
   static new(addr_keyhash: Ed25519KeyHash): ScriptPubkey;
 }
+/**
+ */
+declare export class ScriptRef {
+  free(): void;
+
+  /**
+   * @returns {Uint8Array}
+   */
+  to_bytes(): Uint8Array;
+
+  /**
+   * @param {Uint8Array} bytes
+   * @returns {ScriptRef}
+   */
+  static from_bytes(bytes: Uint8Array): ScriptRef;
+
+  /**
+   * @param {NativeScript} native_script
+   * @returns {ScriptRef}
+   */
+  static new_native_script(native_script: NativeScript): ScriptRef;
+
+  /**
+   * @param {PlutusScript} plutus_script
+   * @returns {ScriptRef}
+   */
+  static new_plutus_script(plutus_script: PlutusScript): ScriptRef;
+}
```

Comment:
Looks good with two constructor functions! 

Please add functions:
- `.is_native_script()`
- `.is_plutus_script()`
- `.native_script()`
- `.plutus_script()`

With getters returning an option, same as in the outputs

Reply:
Okay : )